pyproject.toml

```diff
@@ -67,3 +67,51 @@ select = [
 "**/{tests}/*" = ["B007"]
 "__init__.py" = ["F401"]
 "docs/conf.py" = ["F401"]
+
+[tool.pixi.project]
+name = "jaxsim"
+channels = ["conda-forge"]
+platforms = ["linux-64", "linux-aarch64", "osx-64", "osx-arm64"]
+
+[tool.pixi.dependencies]
+coloredlogs = "*"
+jax = "*"
+jaxlib = "*"
+jaxlie = "*"
+jax-dataclasses = "*"
+pptree = "*"
+rod = "*"
+typing_extensions = "*"
+lxml = "*"
+mediapy = "*"
+mujoco = "*"
+libgz-sim8 = "*"
```

Comment:
Should we specify here the minimum versions as `setup.cfg`?

Reply:
To be honest, I wanted to avoid having that information duplicated. We can probably just add a `pip check` somewhere before the tests to check if we are violating the setup.cfg constraints, and only add the constraint if necessary.